src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom'
+import { getArticleById } from '../api';
+
+const IndividualArticle = () => {
+
+    const { article_id } = useParams();
+
+    const [individualArticle, setIndividualArticle] = useState({})
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        setIsLoading(true)
+        getArticleById(article_id)
+        .then((receivedArticle) => {
+            setIndividualArticle(receivedArticle);
+        })
+        .then(() => {
+            setIsLoading(false);
+        })
+    },[])
+
+
+    return isLoading ? (<h2>Loading article...</h2>) : (
+        <div className="browseArticlesContainer">
```

Comment:
Ideally we should avoid using divs where possible - could we use a more semantic html tag here?